### PR TITLE
review dependencies

### PR DESCRIPTION
**Pull request by fmessmer**
sorting, cleaning up  and review (used roscompile)

Not yet done for cob_teleop see #92 and #93

### CODE REVIEW

File: cob_script_server/CMakeLists.txt [hunk 1]
```diff
@@ -27,50 +23,24 @@ add_service_files(
...
 
-install(DIRECTORY scripts
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-)
-
-install(DIRECTORY msg
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-)
-
-install(PROGRAMS src/cob_console.py
```
Comment (floweisshardt) [inline cob_script_server/CMakeLists.txt:66]: why did you delete this install tags? aren't they needed?

Comment (fmessmer) [inline cob_script_server/CMakeLists.txt:66]: This was a duplicate of L70

**Comment by floweisshardt** [comment]
hard to oversee if anything is missing. I'll merge and give it a try on the buildfarm.
